src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -0,0 +1,96 @@
+module sonic-device_metadata {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-device_metadata";
+    prefix device_metadata;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+        revision-date 2019-07-01;
+    }
+
+    description "DEVICE_METADATA YANG Module for SONiC OS";
+
+    revision 2020-04-10 {
+        description "First Revision";
+    }
+
+    container sonic-device_metadata {
+
+        container DEVICE_METADATA {
+
+            description "DEVICE_METADATA part of config_db.json";
+
+            container localhost{
+
+                leaf hwsku {
+                    type stypes:hwsku;
+                }
+
+                leaf default_bgp_status {
+                    type enumeration {
+                        enum UP;
+                        enum DOWN;
+                    }
+                }
+
+                leaf docker_routing_config_mode {
+                    type enumeration {
+                        enum unified;
+                        enum split;
+                    }
+                }
+
+                leaf hostname {
+                    type string {
+                        length 1..255;
+                    }
+                }
+
+                leaf platform {
```

Comment:
the platform should also go to state db.

Reply:
Same for the platform field.

Reply:
As per https://github.com/Azure/SONiC/wiki/Configuration#device-metadata, we have the platform in allowable config, so resolving it. Let me know if this must be removed.